packages/server/lib/project-base.ts

```diff
@@ -49,6 +49,7 @@ export interface Cfg extends ReceivedCypressOptions {
 const localCwd = process.cwd()
 
 const debug = Debug('cypress:server:project')
+const eDebug = Debug('cypress:server:project:emit')
```

Comment:
Maybe rename this to emitDebug

Reply:
Actually just removing it entirely - probably won't be useful for other things, unless digging into mocha events